From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Viljanen <matti.viljanen@jolla.com>
Date: Mon, 20 Jan 2025 12:06:39 +0000
Subject: [PATCH] Disable memory optimisation in collision test

This works for i486, but fails for armv7hl and aarch64 with Scratchbox.
---
 tests/collisions/main.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/collisions/main.c b/tests/collisions/main.c
index 30aee4a4fb913b589b092ff6e241860ce34d4b39..b8241e3a05772aa2dd08a2283f2c42af216e0635 100644
--- a/tests/collisions/main.c
+++ b/tests/collisions/main.c
@@ -764,6 +764,7 @@ static size_t search_collisions(
             addHashCandidate(hashCandidates, h, htype, nbCandidates++);
         }
     }
+#if 0
     if (nbCandidates < maxNbH) {
         /* Try to mitigate gnuc_quicksort behavior, by reducing allocated memory,
          * since gnuc_quicksort uses a lot of additional memory for mergesort */
@@ -777,6 +778,7 @@ static size_t search_collisions(
                 (unsigned)(tableSize >> 20),
                 (unsigned)((tableSize - (nbCandidates * hashByteSize)) >> 20) );
     }
+#endif
     double const storeTDelay = difftime(time(NULL), storeTBegin);
     DISPLAY(" Stored %llu hash candidates in %s \n",
             (unsigned long long) nbCandidates, displayDelay(storeTDelay));
